src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,2 +1,8 @@
-// Add the required props
-export const UserInfo = () => <>UserInfo markup</>;
+export const UserInfo = ({ user }) => {
+  const {email, name} = user;
+  return(
+
+  <a className="UserInfo" href={`mailto:${email}`}>
```

Comment:
Don't forger about code style
```suggestion
export const UserInfo = ({ user }) => {
  const { email, name } = user;
  
  return (
    <a className="UserInfo" href={`mailto:${email}`}>
```